.travis.yml

```diff
@@ -4,6 +4,7 @@ php:
     - 5.4
     - 5.5
     - 5.6
+    - 7.0
```

Comment:
@jubianchi Added.

Also I fixed standard to 2 spaces (like commands bellow). If you prefer 4 just let me know.